parsl/tests/test_mpi_apps/test_mpiex.py

```diff
@@ -0,0 +1,70 @@
+"""Tests for the wrapper class"""
+from inspect import signature
+from pathlib import Path
+
+import pytest
+
+import parsl
+from .test_mpi_mode_enabled import get_env_vars
+from parsl import HighThroughputExecutor, Config
+from parsl.launchers import SimpleLauncher
+from parsl.providers import LocalProvider
+from parsl.executors.high_throughput.mpi_executor import MPIExecutor
+
+cwd = Path(__file__).parent.absolute()
+pbs_nodefile = cwd.joinpath("mocks", "pbs_nodefile")
+
+
+def local_setup():
+    config = Config(
+        executors=[
+            MPIExecutor(
+                max_workers_per_block=1,
+                provider=LocalProvider(
+                    worker_init=f"export PBS_NODEFILE={pbs_nodefile}",
+                    launcher=SimpleLauncher()
+                )
+            )
+        ]
+    )
+    parsl.load(config)
+
+
+def local_teardown():
+    parsl.dfk().cleanup()
+    parsl.clear()
+
```

Comment:
You can use `local_config()` for the above, instead of `local_setup` and `local_teardown`: return `config` from `local_config()` without calling `parsl.load` yourself, and conftest.py will handle the loading, unloading, and will also handle some extra stuff like directing your run files into `.pytest` - pretty much `local_setup` and `local_teardown` are only needed for tests that *aren't* loading following the: load DFK... test some stuff... shutdown DFK pattern.

Reply:
Done. I also added a little to the CONTRIBUTING.rst document describing this feature